defaults/main.yml

```diff
@@ -13,5 +13,13 @@ letsencrypt_certbot_verbose: true
 letsencrypt_pause_services: []
 # Force Certificate Reneval
 letsencrypt_force_renew: true
-# Additional certbot arguments
-letsencrypt_certbot_args: []
+# certbot arguments
+letsencrypt_certbot_args: ["certonly",
```

Comment:
I think having all flags in a list is an improvement to the current solution. Anyway there are a few parameters that are mandatory for this role to work. Maybe having a variable `letsencrypt_certbot_default_args` that is combined with the user provided ones is easier to use than having to remember all flags when the intention was only to add one.

```yaml
letsencrypt_default_certbot_args:
  - "certonly"
  - "--text""
  ...
```

and within the role

```yaml
- set_fact: _letsencrypt_combined_args="{{ letsencrypt_default_certbot_args + letsencrypt_certbot_args }}"

- set_fact: _letsencrypt_certbot_args="{{ _letsencrypt_combined_certbot_args + ['--renew-by-default'] }}"
  when: letsencrypt_force_renew == true

- set_fact: _letsencrypt_certbot_args="{{ _letsencrypt_combined_certbot_args + ['--keep-until-expiring'] }}"
  when: letsencrypt_force_renew != true
```